FirebaseAuth/Tests/Sample/Sample/MainViewController+Internal.h

```diff
@@ -51,6 +51,8 @@ typedef void (^TestAutomationCallback)(NSError *_Nullable error);
 
 @property(nonatomic) NSURL *actionCodeContinueURL;
 
+@property(nonatomic) NSString *appleNonce;
```

Comment:
Use `copy` for `NSString` properties. go/objc-style#setters-copy-nsstrings

Reply:
Done.